#! /bin/sh /usr/share/dpatch/dpatch-run
## man-wvrtf-name.dpatch by Matej Vela <vela@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: help/man/wvRTF.1: Fix NAME section (#339398).

@DPATCH@
diff -urNad wv-1.2.1~/help/man/wvRTF.1 wv-1.2.1/help/man/wvRTF.1
--- wv-1.2.1~/help/man/wvRTF.1	2001-02-17 22:53:28.000000000 +0100
+++ wv-1.2.1/help/man/wvRTF.1	2006-03-18 11:47:46.000000000 +0100
@@ -1,7 +1,7 @@
 .PU
 .TH wvRTF 1 
 .SH NAME
-wvHtml \- convert msword documents to RTF
+wvRTF \- convert msword documents to RTF
 .SH SYNOPSIS
 .ll +8
 .B wvRTF in_word_doc
